### ISSUE TITLE
Remove deprecated mallinfo() calls

### ISSUE THREAD

**Issue by burghardt** [burghardt/clamfs#60]
On Fedora rawhide ClamFS FTBFS due to deprecated mallinfo() call.
```
stats.cxx: In member function ‘void clamfs::Stats::dumpMemoryStatsToLog()’:
stats.cxx:73:35: error: ‘mallinfo mallinfo()’ is deprecated [-Werror=deprecated-declarations]
   73 |     struct mallinfo mi = mallinfo();
      |                                   ^
In file included from stats.hxx:33,
                 from stats.cxx:25:
/usr/include/malloc.h:118:24: note: declared here
  118 | extern struct mallinfo mallinfo (void) __THROW __MALLOC_DEPRECATED;
      |                        ^~~~~~~~
cc1plus: all warnings being treated as errors
make[2]: *** [Makefile:391: stats.o] Error 1
```

**Comment by burghardt** [comment]
Probably closed by #64.

**Comment by burghardt** [comment]
Fedora rawhide build is fine now.
